install/plane-setup/observability-plane/setup-opensearch-cluster.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+## NOTE
+# Please ensure that any commands in this script are idempotent as the script may run multiple times
+
+# 1. Check OpenSearch cluster status and wait for it to become ready. Any API calls to configure
+#    the cluster should be made only after the cluster is ready.
+
+openSearchHost="${OPENSEARCH_HOST:-http://opensearch:9200}"
+
+echo "Checking OpenSearch cluster status"
+while true; do
+    clusterHealth=$(curl --insecure \
+                         --location "$openSearchHost/_cluster/health" \
+                         --show-error \
+                         --silent)
+    echo $clusterHealth | jq
+    clusterStatus=$(echo "$clusterHealth" | jq --raw-output '.status')
+    if [[ "$clusterStatus" == "green" || "$clusterStatus" == "yellow" ]]; then
+        echo -e "OpenSearch cluster ready. Continuing with setup...\n"
+        break
+    fi
+    echo "Waiting for OpenSearch cluster to become ready..."
+    sleep 20
```

Comment:
Shall we use about 10 seconds? Or an exponential backoff? Having a 20-second interval might delay the install time. For example, if the OpenSesame got ready in 30 seconds, there is another 10 second wait.